app/styles/app.scss

```diff
@@ -36,13 +36,4 @@ body.ember-application{
 
 button {
   @include btn;
-}
-
-a:hover {
-	opacity: 80%;
-}
-
-.c-toggle-box__label {
-  // @todo 	remove when we replace the toggle box component
-  font-family: $font-body;
```

Comment:
I added this code recently -- let's keep it!

Reply:
Sorry, deleted the wrong thing in a merge conflict. Added back.